code-challenges.js

```diff
@@ -2,53 +2,121 @@
+    return `${temp2} degrees Farhenheit is above boiling point.`
+  }
+  else if (temp === 212) {
+    return `${temp3} degrees Farhenheit is at boiling point.`
+  }
+  else {
+    return "undefined"
+  }
+}
+console.log(boilingPt(temp1)); //prints: 35 degrees Farhenheit is below boiling point.
+//----------------------------------------------------------------------------
+// ((2)) Create a function that multiplies each number in the array by 5 using a for loop. Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 var myNumbers1 = [3, 7, 0, 6, -9]
-
-
-
-
-// --------------------3) Create a function that multiplies each number in the array by 5 using map.
-// Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
-
+//Create a funtion named 'multiples' to build a loop that'll multiply every value of the oldArray, "myNumber1" by 5
+//Create a variable to hold the new array of the multiples called "newArr"
+//Create the loop nestled in the function that'll multiply the oldArray by 5 while also pushing the modidified array into the new array using the method .push
+const multiples = (oldArray) => {
+  let newArr = []
+  for (let i=0; i<oldArray.length; i++){
+    newArr.push(oldArray[i] * 5)
+  }
+  return newArr
+}
+console.log (multiples(myNumbers1)) //prints: [15, 35, 0, 30, -45]
+//----------------------------------------------------------------------------
+// ((3)) Create a function that multiplies each number in the array by 5 using map. Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 var myNumbers2 = [8, -7, 0, 6, 2]
-
-
-
-
-// --------------------4) Create a function that takes in an array and returns an array with only the values at odd indexes.
+//Create a function named multiplyByFive and use the map method to iterate the called array, myNumber2
+//Then return the output-value with a multiplication of 5
+  const multiplyByFive = myNumbers2.map(value => {
+    return value *5
+  })
+console.log(multiplyByFive); //prints: [40, -35, 0, 30, 10]
+//----------------------------------------------------------------------------
+// ((4)) Create a function that takes in an array and returns an array with only the values at odd indexes.
 // Use the test variables provided below. Expected output: [ "Chai tea", "Mocha", "Double espresso" ]
-
+//index:              0               1             2            3...
 var coffeeRun = ["Vanilla latte", "Chai tea", "Black coffee", "Mocha", "Americano", "Double espresso"]
-
-
-
-
-// --------------------5) Create a function that removes all the vowels from a string.
+//Declare a function entitled oddIndexExtract for the project
+//Return with a .filter-method using both value and index arguments. Remember that higher order functions can house up to 3 built-in arguments (value, index, array)
+const oddIndexExtract = (oldArray) => {
+  return oldArray.filter((value, index)=> {
+    return index %2 !== 0
+  })
+}
```

Comment:
nice work! I like your pseudocode as well. it's very clear what your approach is.